sql/core/src/test/scala/org/apache/spark/sql/ApproximatePercentileQuerySuite.scala

```diff
@@ -129,7 +144,7 @@ class ApproximatePercentileQuerySuite extends QueryTest with SharedSQLContext {
     withTempView(table) {
       (1 to 1000).toDF("col").createOrReplaceTempView(table)
       checkAnswer(
-        spark.sql(s"SELECT percentile_approx(col, array(0.25 + 0.25D), 200 + 800D) FROM $table"),
+        spark.sql(s"SELECT percentile_approx(col, array(0.25 + 0.25D), 200 + 8000D) FROM $table"),
```

Comment:
here, fix the test case by increasing accuracy

Reply:
I recall that without the change the answer was "499", which is also really close, so I think this is fine.

Reply:
ok, I'll change the expected result to 499